src/router.js

```diff
@@ -112,6 +111,13 @@ const routes = [
     component: VolunteerPartnerSignupView,
     meta: { loggedOutOnly: true }
   },
+  {
+    path: "/referral/:referredByCode",
+    beforeEnter: (to, from, next) => {
+      const referredByCode = to.params.referredByCode;
+      next(`/signup?referral=${referredByCode}`);
```

Comment:
At some point should we update  `/signup?referral=` and `/signup/student/:partnerId` to use `/sign-up` as well?